libraries/Microsoft.Bot.Builder.Ai.QnA/QnAMakerMiddleware.cs

```diff
@@ -11,9 +11,11 @@ namespace Microsoft.Bot.Builder.Ai.QnA
 {
     public class QnAMakerMiddleware : IMiddleware
     {
+        public const string QnAMakerMiddlewareName = "QnAMakerMiddleware";
         public const string QnAMakerResultKey = "QnAMakerResult";
         public const string QnAMakerTraceType = "https://www.qnamaker.ai/schemas/trace";
         public const string QnAMakerTraceLabel = "QnAMaker Trace";
+        public const string Obfuscated = "****";
```

Comment:
Add a comment describing what/why this is there. 

Seeing this sets off all sorts of red flags.